packages/configuration-validation/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 1.1.0
```

Comment:
why two versions added?

Reply:
it looks like `1.0.0` was missing from changelog